package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +315,20 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Getting element information from element column.
+        if all(elements):
+            element_set = set(i.capitalize() for i in set(elements))
+            if all(element in SYMB2Z for element in element_set):
+                attrs.append(Elements(np.array(
+                             [i.capitalize() for i in elements],
+                             dtype=object)))
+            else:
+                warnings.warn("Invalid elements found in the PDB file, "
+                              "elements attributes will not be populated.")
+        else:
+            warnings.warn("Element information is absent or missing for a few "
+                          "residues. Elements attributes will not be populated.")
```

Comment:
@AmeyaHarmalkar if you are fixing @zemanj's comment: https://github.com/MDAnalysis/mdanalysis/pull/2627#discussion_r399829470 then I probably would also change `a few residues` here to `some atoms` (since elements are atom-wise).

Reply:
Sure thing. Updating and pushing through